Makefile.ide

```diff
@@ -106,7 +106,7 @@ $(COQIDEBYTE): $(LINKIDE)
 	$(HIDE)$(OCAMLC) $(COQIDEFLAGS) $(BYTEFLAGS) -o $@ unix.cma threads.cma lablgtk.cma \
 	        lablgtksourceview2.cma str.cma $(IDEFLAGS) $(IDECDEPSFLAGS) $^
 
-ide/coqide_main.ml: ide/coqide_main.ml4 config/Makefile # no camlp4deps here
+ide/coqide_main.ml: ide/coqide_main.mli ide/coqide_main.ml4 config/Makefile # no camlp4deps here
```

Comment:
Does the generated ml really depend on the mli?

Reply:
No, that must be a mistake. Will fix.